.github/workflows/kind-e2e.yaml

```diff
@@ -47,7 +47,9 @@ jobs:
     env:
       GOPATH: ${{ github.workspace }}
       GO111MODULE: off
-      KO_DOCKER_REPO: kind.local
+      REGISTRY_PORT: 5000
+      REGISTRY_NAME: registry.local
```

Comment:
there was a bit of discussion that landed us here, want to add a comment?

Reply:
Done